### PR TITLE
add python3-xmltodict-pip package

### PR DESCRIPTION
**Pull request by hakuturu583**
Signed-off-by: Masaya Kataoka <ms.kataoka@gmail.com>

xmltodict Packages in rosdistro is Python2 version.
https://packages.ubuntu.com/search?keywords=python-xmltodict&searchon=names&suite=groovy&section=all

We want to use it with Python3, so we want to add python3-xmltodict-pip package to rosdistro.

### CODE REVIEW

**Comment by hakuturu583** [comment]
Sorry, I cannot understand why CI was failed.
Does anyone has idea about it?

File: rosdep/python.yaml [hunk 1]
```diff
@@ -5239,7 +5239,7 @@ python-wrapt:
   ubuntu: [python-wrapt]
 python-ws4py:
   debian: [python-ws4py]
-  ubuntu: [python-ws4py]
+  ubuntu: [python-ws4py]python3-w
```
Comment (tfoote) [inline rosdep/python.yaml:5242]: CI is failing due to the syntax error here.

**Review by tfoote — CHANGES_REQUESTED** [review]
Please use the available natively packaged versions not the pip version. I found a python3 equivalent at: https://packages.ubuntu.com/search?keywords=python3-xmltodict

From the contributing guide we strongly prefer native packages so that they can be packaged against.

**Comment by hakuturu583** [comment]
changed to native packages.

**Comment by hakuturu583** [comment]
Ubuntu : https://packages.ubuntu.com/search?keywords=python3-xmltodict
Arch : https://www.archlinux.org/packages/community/any/python-xmltodict/
Alpine：https://pkgs.alpinelinux.org/package/edge/community/x86/py3-xmltodict
Debian：https://packages.debian.org/search?keywords=python3-xmltodict

**Review by tfoote — APPROVED** [review]
Thanks for the iteration. It looks good to me now.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -6874,6 +6874,11 @@ python3-wxgtk4.0:
   fedora: [python3-wxpython4]
   gentoo: [dev-python/wxpython]
   ubuntu: [python3-wxgtk4.0]
+python3-xmltodict:
+  alpine: [py3-xmltodict]
+  arch: [python-xmltodict]
+  debian: [python3-xmltodict]
+  ubuntu: [python3-xmltodict]
```
Comment (clalancette) [inline rosdep/python.yaml:6881]: Can you also add a fedora entry, like:

```
fedora: [python3-xmltodict]
```

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rosdep/python.yaml (hunk 2 above)
Comment (sloretz) [reply rosdep/python.yaml:6881]: https://fedora.pkgs.org/32/fedora-x86_64/python3-xmltodict-0.12.0-7.fc32.noarch.rpm.html

Comment (hakuturu583) [reply rosdep/python.yaml:6908]: I added fedra package in https://github.com/ros/rosdistro/pull/27499/commits/404adf31b1185b08a0f6098d5aef3979c174f99f

**Review by clalancette — APPROVED** [review]

**Pull request merged**